backend/Pipfile

```diff
@@ -44,6 +44,8 @@ uritemplate = "*"
 uwsgi = "*"
 ddt = "*"
 tqdm = "*"
+nose = "*"
```

Comment:
What's nose being used for? It looks like you're just using Django's built-in test runner here

Reply:
I was getting an error when running tests locally for some reason, it said nose wasn't installed. Adding this fixed that. :/

Reply:
ddt needs it, but doesn't include it in its dependencies, apparently.